pages/index.tsx

```diff
@@ -18,6 +18,7 @@ export default function Home({
     manufacturers: TestManfacturers[];
   };
 }) {
+  const [value, setValue] = useState<Date | null>(null);
```

Comment:
We want to be able to share these values when we eventually do the mapping for the tests, so we pull it out and then pass it through